src/main/java/com/mraof/minestuck/world/lands/LandTypes.java

```diff
@@ -137,7 +137,8 @@ public static void registerTitle(final RegistryEvent.Register<TitleLandType> eve
 		
 		registry.register(new NullTitleLandType().setRegistryName("null"));
 		registry.register(new FrogsLandType().setRegistryName("frogs"));
-		registry.register(new WindLandType().setRegistryName("wind"));
+		registry.register(new WindLandType(WindLandType.Variant.WIND).setRegistryName("wind"));
+		//registry.register(new WindLandType(WindLandType.Variant.WINDMILLS).setRegistryName("windmills"));
```

Comment:
Some of this work was left around to save unfinished progress and show future intentions. The windmill WindLandType was left here because I did not have enough work done to add it but wanted to keep it around

Reply:
Alright. My suggestion is then to remove it and readd it to a new branch, unless you're planning on finishing it before this pull request is merged. (In the scenario that it isn't implemented anytime soon, I'd rather have it separated from the main codebase until it is)

Reply:
Removed!